_data/health.yml

```diff
@@ -64,6 +64,12 @@ websites:
       software: Yes
       doc: http://windows.microsoft.com/en-au/windows/two-step-verification-faq
 
+    - name: MedicAlert
+      url: https://www.medicalert.ca
+      img: medicalert.png
+      twitter: medicalertCA
```

Comment:
please change to `MedicAlertCA`.
After that then this PR is also ready to be merged.